docs/spin.toml

```diff
@@ -5,13 +5,6 @@ description = "The Spin documentation website running on... Spin."
 authors = [ "Fermyon Engineering <engineering@fermyon.com>" ]
 trigger = { type = "http", base = "/" }
 
-[[component]]
-source = "modules/bartholomew.wasm"
-id = "bartholomew"
-files = [ "content/**/*" , "templates/*", "scripts/*", "config/*"]
-[component.trigger]
-route = "/..."
-
 [[component]]
```

Comment:
The download is just the install script, which we are also moving, so we can remove this as well, unless we cannot redirect those requests.

Reply:
Fixed